change/@itwin-tree-widget-react-3c0d5f0d-6322-4687-b98c-a090d4527382.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fixed `ModelsTree` and `CategoriesTree` not applying custom hierarchy level size limit.",
```

Comment:
is it just those trees or others too?

Reply:
Only those two as they were using `VisibilityTree` component. Others used `Tree` which correctly handled hierarchy level size config.